testing/src/main/java/io/stargate/it/http/RestUtils.java

```diff
@@ -74,22 +75,18 @@ public static String postWithHeader(
       Headers headers, String path, String requestBody, int expectedStatusCode) throws IOException {
     OkHttpClient client = client();
 
-    Request request;
+    RequestBody rb =
+        Optional.ofNullable(requestBody)
```

Comment:
nit: Use of `Optional` seems unnecessary.  Any reason not to use `if (requestBody != null)` ?

Reply:
Done.